cognite/powerops/transformations/transformations.py

```diff
@@ -0,0 +1,397 @@
+    def load(cls, transformation_: dict[str, Any]) -> "Transformation":
+        # sourcery skip: move-assign
+        (transformation_name,) = transformation_
+        class_index = [cls.__name__ for cls in Transformation.__subclasses__()].index(transformation_name)
+        if transformation_input := transformation_[transformation_name]["input"]:
+            return Transformation.__subclasses__()[class_index].__call__(**transformation_input)
+        else:
+            return Transformation.__subclasses__()[class_index].__call__()
+
+    @abc.abstractmethod
+    def apply(self,
+              time_series_data: pd.Series,
+              ):
+        ...
+
+class Add(Transformation):
+    transformation_name = "add" # using the class name instead or keep?
+    value: float
+
+    def apply(self,
+              time_series_data: pd.Series,
+              ):
+        return time_series_data + self.value
+
+
+class Multiply(Transformation):
+    transformation_name = "multiply"
+    value: float
+
+    def apply(self,
+              time_series_data: pd.Series,
+              ):
+        return time_series_data * self.value
+
+
+class StaticValues(Transformation):
+    transformation_name = "static_values"
+    shift_minutes: int = 0  # This is given at runtime - have a set method for this? looks like default value of 0 is always used
+    relative_datapoints: list[RelativeDatapoint]
+    _start: Optional[int] = None
+
+    @property
+    def start(self) -> int:
+        return self._start
+
+    def set_start(self, value: int) -> None:
+        self._start = value
+
+    def apply(self, _: pd.Series) -> pd.Series:
+        if not self.start:
+            raise ValueError("start time has not been set")
+        return _relative_datapoints_to_series(self.relative_datapoints, ms_to_datetime(self.start), self.shift_minutes)
+
+
+class ToBool(Transformation):
+    transformation_name = "to_bool"
+
+    def apply(self, time_series_data: pd.Series) -> pd.Series:
+        return (time_series_data > 0).astype(int)
+
+
+class ZeroIfNotOne(Transformation):
+    transformation_name = "zero_if_not_none"
+
+    def apply(self, time_series_data: pd.Series) -> pd.Series:
+        return (time_series_data == 1).astype(int)
+
+
+class OneIfTwo(Transformation):
+    transformation_name = "one_if_two"
+
+    def apply(self, time_series_data: pd.Series) -> pd.Series:
+        return (time_series_data == 2).astype(int)
+
+
+class HeightToVolume(Transformation):
+    transformation_name = "height_to_volume"
+    _volumes: Optional[list[float]] = None
+    _heights: Optional[list[float]] = None
+
+    @property
+    def volumes(self) -> list[float]:
+        return self._volumes
+
+    @property
+    def heights(self) -> list[float]:
+        return self._heights
+
+    # TODO: run this function in preprocessor before running apply. Need a check on transformation name
+    def set_and_prepare_volumes(self, model: dict, object_type: str, instance: str | int):
```

Comment:
I think maybe you should have an optional `init` method in the base class to cover this case.